packages/actor-function-factory-term-lesser-than/test/op.lesserThan-test.ts

```diff
@@ -148,24 +176,65 @@ describe('evaluation of \'<\'', () => {
     });
   });
 
+  describe('with RDF literal operands like', () => {
```

Comment:
```suggestion
  describe('with literals of unknown types like', () => {
```

(Also check whether these tests are positioned close to the tests for [subtype substitution](https://www.w3.org/TR/xpath20/#dt-subtype-substitution))